codeQuality/index.md

```diff
@@ -0,0 +1,173 @@
+# Code Quality
+
+## Linting / Prettier
+
+In the CDapp we have linting rules enabled. When commiting, a number of pre-commit checks will run to ensure your code is consistent with these.
+
+Many of us also use Prettier to ensure that our code is formatted neatly on the page
```

Comment:
This will be a imposed rule, once we get prettier wired in, so this line should state that fact